hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -389,6 +389,14 @@
                  "+ext_clk_type=ExtClkLowSpeed", "+use_extclk=1", "+extclk_low_speed_sel=1"]
       reseed: 5
     }
+    {
+      name: chip_sw_i2c_host_tx_rx
+      uvm_test_seq: chip_sw_i2c_tx_rx_vseq
+      sw_images: ["//sw/device/tests/sim_dv:i2c_tx_rx_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: []
+      reseed: 1
```

Comment:
perhaps remove empty `run_opts` and increase reseed since we randomize the number and value of date

Reply:
ah yes sounds good, let me do that. Should i just leave reseed out also since we are defaulting to 3 now?

Reply:
yeah, we could just use default reseed